src/widget/wsearchlineedit.h

```diff
@@ -45,11 +49,16 @@ class WSearchLineEdit : public QLineEdit, public WBaseWidget {
     void slotClearSearch();
     bool slotClearSearchIfClearButtonHasFocus();
 
+    /// Select the next/previous entry from the history
```

Comment:
The comment is misleading and wrong.

- The function selects an entry relative to the currently selected entry in the history and executes the search
- The parameter specifies the distance in *steps* (positive/negative = downward/upward)